css/src/components/gradient-card.scss

```diff
@@ -0,0 +1,11 @@
+.gradient-card {
+	border: $border-width-md solid $border !important;
```

Comment:
Does the `!important` flag really need to be used here? Please remove them all.

Reply:
No turns out it can be over-ridden by atomics, which defeats the purpose but the important here doesn't stop that because it's the same specificity but listed before atomics so I removed it.

Reply:
Same was true of the remaining, they've all been removed.